wait/all.go

```diff
@@ -62,6 +62,13 @@ func (ms *MultiStrategy) WaitUntilReady(ctx context.Context, target StrategyTarg
 	}
 
 	for _, strategy := range ms.Strategies {
+		if strategy == nil {
```

Comment:
bug: this won't detect a nil type which implements Strategy see https://go.dev/play/p/yz46T3tUXOR